libraries/standard/mqtt/cbmc/proofs/MQTT_GetIncomingPacketTypeAndLength/MQTT_GetIncomingPacketTypeAndLength_harness.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Insert copyright notice
+ */
```

Comment:
I think the copyright stuff is required on all these files before putting into open source.

```
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
```

Reply:
I did not add it yet as the license for our code is not finalized yet. I can add this in temporarily.